src/test/java/com/fundation/webservice/model/CriteriaConvertTest.java

```diff
@@ -0,0 +1,25 @@
+package com.fundation.webservice.model;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class CriteriaConvertTest {
+    CriteriaConvert criteriaConvert = new CriteriaConvert();
+
+    @Before
+    public void setUp(){
+        criteriaConvert.setSrcPath(Directories.RSRC_DIR.getDir());
+        criteriaConvert.setDestPath(Directories.RSRC_DIR.getDir());
+    }
+    @Test
```

Comment:
It might be better to insert an additional line to space between both methods.

Reply:
Okay, it's fixed.